src/main/java/teammates/ui/webapi/GetFeedbackSessionLogsAction.java

```diff
@@ -91,10 +107,11 @@ public JsonResult execute() {
         feedbackSessions.forEach(fs -> sessionsMap.put(fs.getFeedbackSessionName(), fs));
 
         fsLogEntries = fsLogEntries.stream().filter(logEntry -> {
-            String fslType = logEntry.getFeedbackSessionLogType();
-            FeedbackSessionLogType convertedFslType = FeedbackSessionLogType.valueOfLabel(fslType);
-            if (convertedFslType == null) {
-                // If the feedback session log type retrieved from the log is invalid, ignore the log
+            String logType = logEntry.getFeedbackSessionLogType();
+            FeedbackSessionLogType convertedLogType = FeedbackSessionLogType.valueOfLabel(logType);
+            if (convertedLogType == null || fslTypes != null && !convertedFslTypes.contains(convertedLogType)) {
```

Comment:
The filtering should be done in Cloud Logging itself, not here. But it doesn't really matter now, as the backend is going to be reworked anyway.